extension/view/js/translation-notification-fxtranslations.js

```diff
@@ -169,24 +135,6 @@ window.MozTranslationNotification = class extends MozElements.Notification {
     return this.querySelector(`[anonid=${anonId}]`);
   }
 
-  fromLanguageChanged() {
-    this.translationNotificationManager.reportInfobarEvent("change_lang");
```

Comment:
where do we report these infobar events now?

Reply:
Actually if you look carefully, you'll notice that this function was already part of the dead code that I refer to, and it wasn't being called at all. Maybe we should manually triple check in the telemetry backend if all events are actually being sent (from everywhere and not just here). I attached this function to the dropdown event and is not being called.